app/views/entities/index.html.erb

```diff
@@ -18,6 +17,6 @@
   <% end %>
   </div> 
 </div>
-<div class="btn_dev1 fixed-bottom">
+<div class="btn_dev1 fixed-bottom p-3">
```

Comment:
- [ ] @AhmadiGu great work implementing the changes requested however as previously requested kindly ensure the design matches the Behance design given, this page is supposed to list all transactions for a given category following the design given, the button you talking about lists all transactions.
![image](https://user-images.githubusercontent.com/61408860/232228822-5f08340b-fa67-4d7e-8750-679fedec6453.png)

![image](https://user-images.githubusercontent.com/61408860/232228728-c21b7e73-554a-4c7b-9bbb-a9c0ef9f13c4.png)